src/corehost/build.sh

```diff
@@ -155,6 +156,13 @@ case $__build_arch in
 esac
 __cmake_defines="${__cmake_defines} ${__arch_define}"
 
+# Configure environment if we are doing a cross compile.
```

Comment:
@gkhanna79 If ROOTFS_DIR is not set in coreclr or corefx, the default path is set in cross/rootfs/${build_arch}. This is a patch to build core-setup without having to set ROOTFS_DIR separately when user creates rootfs by build-rootfs.sh script.